### ISSUE TITLE
Tweak bot settings in young wolves arc

### ISSUE THREAD

**Issue by AaronGullickson** [AaronGullickson/mekhq#41]
In many scenarios, the bot is overly timid. She hangs back in the face of clan firepower and gets picked off one by one. Switching to something more like Default V2 Braver may help. However, I also may want to increase piloting caution in scenarios with a lot of hazards like pavement  or ice.

**Comment by AaronGullickson** [comment]
One thing I am noting is that the so called "braveryIndex" is actually a "cowardiceIndex" - high values indicate less bravery. This was poorly named.

**Comment by Thom293** [comment]
Yes.  It's extremely confusing. I did try tweaking that number when I was working on the new settings and didn't seem to do anything either way though.  So I "fixed" it by changing by changing Bersek instead. (Which is how close she wants to be to enemy).

**Comment by AaronGullickson** [comment]
This seemed to work pretty well for Cutting Teeth scenario:

```xml
<behaviorSettings>
        <name>BASIC_CLAN</name>
        <forcedWithdrawal>true</forcedWithdrawal>
        <goHome>false</goHome>
        <autoFlee>false</autoFlee>
        <fallShameIndex>3</fallShameIndex>
        <hyperAggressionIndex>7</hyperAggressionIndex>
        <selfPreservationIndex>3</selfPreservationIndex>
        <herdMentalityIndex>5</herdMentalityIndex>
        <braveryIndex>2</braveryIndex>
       <strategicTargets/>
</behaviorSettings>
```

**Comment by Thom293** [comment]
Looks great!  Adding a "Basic Clan" setting to the main MM repo would be a good thing too, IMHO.

My only comment would be I would change fall shame to 5.  3 is the old default, and on pavement maps she will still kill herself by falling before she ever engages.  5 is the updated scale and it prevents her from taking most move psrs before combat.  She will still take them in combat though.  I updated the Default v2 Braver to that scale last night.

**Comment by AaronGullickson** [comment]
Addressed in f88910f283.
